content/blog/wingingit/winging-it-16.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Debugging CSS Anchor Positioning
+episode: 16
+tags:
+  - CSS
+  - Anchor Positioning
+author:
+  - miriam
+  - james
+date: 2025-02-20
+length: 65 mins
+image:
+  src: winging-it/winging-it-16.jpg
+media:
+  - youtube: 3vwruYb9du4
+summary: |
+  It is frustrating
+  to track down
+  why an anchor isn’t being found.
+  James and Miriam talk
+  with [Tab Atkins-Bittner](https://github.com/tabatkins)
+  about ways to troubleshoot
+  CSS Anchor Positioning issues,
+  and dive into the details
+  to get a better understanding
+  of how Anchor Positioning works.
+---
+
+{% import "embed.macros.njk" as embed %}
+{% import "utility.macros.njk" as utility %}
+
+{{ embed.figure(
+  data=media
+) }}
+
+{{ utility.main_action(
+  'Subscribe to Channel »',
+  subscribe_url
+) }}
+
+ ## What we cover:
+
+- Intro to Anchor positioning demo
+- Containing blocks
+- Element or valid pseudo element
+- Absolute positioned anchor - dom order matters
+- Dialogs and popovers
+- Styles in the same tree
+- Styles actually present
+- Anchor Scope tree
+- Recommendation for success
+- Ways to improve users' understanding of containing blocks rules
+- Coming soon for Anchor Positioning
+
+## Links:
+
+- [Here’s Why Your Anchor Positioning Isn’t Working](https://www.oddbird.net/2025/01/29/anchor-position-validity/)
+- [Xanthir.com](https://xanthir.com/talks/2024-06-07/)
```

Comment:
Can we give this a more descriptive title?

Reply:
That would be good, yes. When I go to this link from the Show Note outline, all I see is Tab's name, a couple social media links, and a link to talks that doesn't appear to work. Do you know more about this link, James?

Reply:
It's slides from a talk -- the arrow keys seem to advance it for me.

Reply:
Updated. There's not a "title" to the talk beyond "Anchor Positioning"